Sources/BBCode/Spoiler2.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines https://www.simplemachines.org
+ * @copyright 2025 Simple Machines and individual contributors
+ * @license https://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version 3.0 Alpha 3
+ */
+
+declare(strict_types=1);
+
+namespace SMF\BBCode;
+
+use SMF\Lang;
+
+/**
+ * Represents the unparsed_equals version of the spoiler BBCode.
+ *
+ * For backward compatibility with old spoiler BBC mods, this BBCode renders
+ * `[spoiler="foo"]bar[/spoiler]` like `[details summary="foo"]bar[/details]`
+ */
+class Spoiler2 extends BBCode
+{
+	/*******************
+	 * Public properties
+	 *******************/
+
+	/**
+	 *
+	 */
+	public string $tag = 'spoiler';
+
+	/**
+	 *
+	 */
+	public ?string $type = 'unparsed_equals';
```

Comment:
Should this be `BBCode::TYPE_UNPARSED_EQUALS` instead?

Reply:
Yes it should. Good catch.\!